chitterView.js

```diff
@@ -0,0 +1,34 @@
+class ChitterView {
+  constructor(model, api) {
+    this.api = api;
+    this.model = model;
+    this.peepContainerEl = document.querySelector('#peep-container');
+    this.peepButtonEl = document.querySelector('#add-peep-button');
+
+    this.peepButtonEl.addEventListener('click', () => {
+      const newPeep = document.querySelector('#peep-input').value;
+      
+      this.api.createPeep(newPeep, (data) => {
+        this.addNewPeep(newPeep)
+      })
+    })
+  }
+
+  displayPeeps() {
```

Comment:
I think this method is not working the way you intended, because I see a list of `[object Object]` instead of the actual peeps. I can see in the network tab that you're correctly fetching the peeps from the API, but I think this method should have `peeps` as argument to then be able to do something with the raw data.